saleor/userprofile/i18n.py

```diff
@@ -70,13 +77,24 @@ class AddressForm(forms.ModelForm):
         ('email', 'email')
     )
 
+
     class Meta:
         model = Address
-        exclude = []
+        exclude = ['phone']
+
+    phoneprefix = ChoiceField(choices=phone_prefixes)
+    phone = forms.CharField()
```

Comment:
I have tried use this one:
https://github.com/stefanfoulis/django-phonenumber-field/blob/master/phonenumber_field/formfields.py
but with not much success, as it wants "correct" number (with phone country code included), while here we want to separate things.

Reply:
No, we don't necessarily want to separate them as fields, we want separate widgets.